01_fibonacci/README.md

```diff
@@ -0,0 +1,22 @@
+# Lab 1 - Fibonacci
+
+Create an executable go program in directory 01_fib/USERNAME
```

Comment:
this says `01_fib` but the readme is in `01_fibonacci`. please fix one or the other.

Reply:
Oh, thats where they're located